core/src/main/scala/com/pingcap/tispark/utils/TiUtil.scala

```diff
@@ -243,4 +244,20 @@ object TiUtil {
 
     mutableRow
   }
+
+  def extractDAGReq(df: DataFrame): TiDAGRequest = {
```

Comment:
what if there are more than one or no `TiDAGRequest` in `df` ? show we return `List<TiDAGRequest>` ?

Reply:
We should only have one TiDAGRequest.

Reply:
maybe zero ?

Reply:
Originally, it is zero while IntelliJ gives me a warning.